topydo/lib/Filter.py

```diff
@@ -214,3 +214,35 @@ def match(self, p_todo):
 
         return False
 
+PRIORITY_MATCH = r"\((?P<operator><=?|>=?|=|!)?(?P<priority>[A-Z]{1})\)"
```

Comment:
The first part can be put in a separate regexp, e.g. `OPERATOR_MATCH`, that can be used by this match expression and the `ORDINAL_TAG_MATCH`.